### PR TITLE
feat: useCustomerReturns

### PR DESCRIPTION
**Pull request by justyna-13**
### 🔗 Linked issue

https://vsf.atlassian.net/browse/SV-119

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSDoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

composable for customer returns 

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/web/composables/useCustomerReturns/useCustomerReturns.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { toRefs } from '@vueuse/shared';
+import type {
+  UseCustomerReturnsReturn,
+  UseCustomerReturnsState,
+  FetchCustomerReturns,
+} from '~/composables/useCustomerReturns/types';
+
+/**
+ * @description Composable managing address data
```
Comment (Szymon-dziewonski) [inline apps/web/composables/useCustomerReturns/useCustomerReturns.ts:9]:
```suggestion
 * @description Composable managing returns data
```

File: apps/web/composables/useCustomerReturns/useCustomerReturns.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ * @example
+ * const { data, loading, fetchCustomerReturns } = useCustomerReturns();
+ */
+export const useCustomerReturns: UseCustomerReturnsReturn = () => {
+  const state = useState<UseCustomerReturnsState>(`useCustomerReturns`, () => ({
+    data: [],
+    loading: false,
+  }));
+
+  /** Function for fetching address data
```
Comment (Szymon-dziewonski) [inline apps/web/composables/useCustomerReturns/useCustomerReturns.ts:20]:
```suggestion
  /** Function for fetching returns data
```

Comment (justyna-13) [reply apps/web/composables/useCustomerReturns/useCustomerReturns.ts]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/112/commits/f50f9b5414c7864375c3cd0beb81d34a55db8e9d

File: apps/web/composables/useCustomerReturns/useCustomerReturns.ts (hunk 1 above)
Comment (justyna-13) [reply apps/web/composables/useCustomerReturns/useCustomerReturns.ts]: https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/112/commits/f50f9b5414c7864375c3cd0beb81d34a55db8e9d

**Review by Szymon-dziewonski — APPROVED** [review]

**Pull request merged**